sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/models/CreateEcKeyOptions.java

```diff
@@ -11,23 +11,23 @@
 import java.util.Arrays;
 import java.util.Map;
 
-public class EcKeyCreateOptions extends KeyCreateOptions {
+public class CreateEcKeyOptions extends CreateKeyOptions {
```

Comment:
Be sure to include appropriate annotations, e.g. `@Fluent`

Reply:
added.